lib/testPaymentExpiry.js

```diff
@@ -0,0 +1,191 @@
+'use strict'
+const _ = require('lodash')
+const moment = require('moment')
+const request = require('co-request')
+const UnacceptableExpiryError = require('../errors/unacceptable-expiry-error')
+
+module.exports = function * (config, payment) {
+  let tester = new TransferTester(config, payment)
+  yield tester.loadCaseExpiries()
+  return tester
+}
+
+function TransferTester (config, payment) {
+  this.config = config
+  this.source_transfers = payment.source_transfers
```

Comment:
Yeah, this is a tricky one. I think probably the best option is to have a single module that does camel to snake conversion for all requests/responses so that we can use camel case everywhere else in the code while still only having to search for one thing.